source/content/guides/decoupled/overview/04-site.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Pantheon Front-End Sites
+subtitle: Site Creation
+description: Create and configure a Front-End Site.
+tags: [webops, workflow, decoupled]
+contributors: [joa-pan, backlineint, cobypear, hckia]
+layout: guide
+showtoc: true
+permalink: docs/guides/decoupled/overview/site
+anchorid: site
+reviewed: "2023-03-16"
+contenttype: [guide]
+innav: [false]
+categories: [create]
+cms: [decoupled]
+audience: [development]
+product: [decoupled]
+integration: [--]
+---
+
+This section provides configuration information for Front-End Sites.
+
+## Prerequisites
+
+* The **Decoupled** tab is enabled in your dashboard after onboarding.
+
+* Your site repository is in GitHub.
+
+* You have a domain to use for the Front-End site.
+
+* You will use Next.js or Gatsby as the frontend framework.
+
+* (Optional). You are using the latest version of Drupal, or WordPress for the CMS.
+
+* (Optional). If using a pre-configured starter kit template, you will use one of the following combinations:
+
+    * [Drupal and Next.js](/guides/decoupled/drupal-nextjs-frontend-starters/)
+
+    * [WordPress and Next.js](/guides/decoupled/wp-nextjs-frontend-starters/)
+
+    * [WordPress and Gatsby](/guides/decoupled/wp-gatsby-frontend-starters/)
+
+    > **NOTE**: Other starter kit configurations are not currently supported.
+
+
+### Compatibility
+
+You can create an application from scratch by importing a repository or you can use the Pantheon starter kit templates to develop a Front-End Site.
+
+You can also create a site without a CMS. This is done by importing the repository and using Next.js or Gatsby as the static site generator.
+
+Supported Versions:
+
+|  Tooling   |      Version    |
+|  :---:     |       :---:     |
+|  Gatsby    |    4 or higher  |
+| Next.js    |   12 or higher  |
+|  Drupal    |   9 or higher   |
+| WordPress  |   5.9 or higher |
+
+## Create a Front-End Site
+
+Select the backend or frontend guide below that meets your configuration requirements and follow the instructions carefully.
+
+### Backend Starters
+
+- [Drupal Backend Starter](/guides/decoupled/drupal-backend-starters)
+- [WordPress Backend Starter](/guides/decoupled/wp-backend-starters)
+
+### Frontend Starters
+
+- [Drupal + Next.js Frontend Starter](/guides/decoupled/drupal-nextjs-frontend-starters)
+- [WordPress + Next.js Frontend Starter](/guides/decoupled/wp-nextjs-frontend-starters)
+- [WordPress + Gatsby Frontend Starter](/guides/decoupled/wp-gatsby-frontend-starters)
+
+### No Starter
+
+- [Create a Front-End Site without a Starter](/guides/decoupled/no-starter-kit)
```

Comment:
I could see this section as its own page, and instead of linking to the guides, putting the guides in a tabbed section.